hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ContainerReplicaCount.java

```diff
@@ -49,6 +49,8 @@ public interface ContainerReplicaCount {
   boolean isSufficientlyReplicatedForOffline(DatanodeDetails datanode,
       NodeManager nodeManager);
 
+  boolean isUnderReplicated();
```

Comment:
I am not sure we really need another method on the interface which is just `!isSufficientlyReplicated()` - we can just call `!isSufficientlyReplicated()` without adding more to the interface.